lib/evm/opFns.ts

```diff
@@ -978,6 +980,69 @@ function updateSstoreGas(runState: RunState, found: any, value: Buffer) {
       }
     }
     return runState.eei.useGas(new BN(runState._common.param('gasPrices', 'netSstoreDirtyGas')))
+  } else if (runState._common.hardfork() === 'istanbul') {
```

Comment:
This very likely has to be a `gte` comparison?

Reply:
Good catch, fixed